src/creator.coffee

```diff
@@ -202,12 +205,14 @@ Hangman.controller 'HangmanCreatorCtrl', ['$timeout', '$scope', '$sanitize', 'Re
 		$scope.attempts = ~~qset.options.attempts or 5
 		$scope.partial = qset.options.partial
 		$scope.random = qset.options.random
+		$scope.enableQuestionBank = qset.options.enableQuestionBank
```

Comment:
Javascript/coffeescript/angular is letting you off easy here, but remember what I said about backwards compatibility: we can't always assume qsets will have these values, since every existing GTP qset saved prior to this feature won't have `options.enableQuestionBank` and `options.questionBankVal`. Instead of throwing a console error, it's just setting the `$scope` values to `undefined`. This is problematic later on when `undefined` values are saved instead of `true` or `false`.